src/sanitizer/cli.py

```diff
@@ -0,0 +1,72 @@
+import asyncio
+from functools import wraps
+
+import click
+
+from .rds import (
+    cleanup,
+    create_instance,
+    create_snapshot,
+    delete_old_snapshots,
+    get_latest_snapshot,
+    restore_snapshot,
+    rotate_password,
+    share_snapshot,
+)
+from .settings import settings
+from .sql import sanitize
+
+
+def async_command(func):
+    @wraps(func)
+    def wrapper(*args, **kwargs):
+        return asyncio.run(func(*args, **kwargs))
+
+    return wrapper
+
+
+@click.command()
+@async_command
+async def main():
+    click.echo("#################### Finding latest snapshot ####################")
+    snapshot = get_latest_snapshot(settings.rds_cluster_id)
+    click.echo(f"Latest snapshot: {snapshot}")
+    click.echo("")
+
+    click.echo("#################### Restoring snapshot ####################")
+    temp_cluster = restore_snapshot(snapshot)
+    click.echo(f"Temporary cluster: {temp_cluster}")
+    click.echo("")
+
+    click.echo("#################### Rotating password ####################")
+    ssm_param, temp_cluster = rotate_password(temp_cluster)
+    click.echo(f"SSM parameter name: {ssm_param}")
+    click.echo("")
+
+    click.echo("#################### Creating instance ####################")
+    temp_instance = create_instance(temp_cluster)
+    click.echo(f"Temporary instance: {temp_instance}")
+    click.echo("")
+
+    click.echo("#################### Sanitizing ####################")
+    await sanitize(temp_cluster, ssm_param)
+    click.echo("")
+
+    click.echo("#################### Creating sanitized snapshot ####################")
+    sanitized_snapshot = create_snapshot(temp_cluster)
+    click.echo(f"Sanitized snapshot: {sanitized_snapshot}")
+    click.echo("")
+
+    click.echo("#################### Creating shared snapshot ####################")
+    shared_snapshot = share_snapshot(sanitized_snapshot)
```

Comment:
The shared snapshot is always created even if `SANITIZER_SHARE_ACCOUNT_IDS` is not set?

Reply:
Right, will make the step optional 🙏